schemas/mod-entities-links/instanceLinkDtoCollection.json

```diff
@@ -51,8 +41,6 @@
   "required": [
     "authorityId",
     "authorityNaturalId",
-    "instanceId",
-    "bibRecordTag",
-    "bibRecordSubfields"
+    "instanceId"
   ]
```

Comment:
Shouldn't `linkingRuleId` be reqiured?

Reply:
Oh, yes, let me add it to required fields' list